FetchMigration/README.md

```diff
@@ -1,74 +1,76 @@
+The tool consists of three components:
+
+* The **Metadata migration** module handles metadata comparison between the source and target clusters. This can output a human-readable report as well as a Data Prepper `pipeline.yaml` file.
+* The **Migration monitor** module monitors the progress of the migration and shuts down the Data Prepper pipeline 
 once the target document count has been reached
-* An "orchestrator" module that sequences these steps as a workflow and manages the kick-off of the Data Prepper 
-process between them.
+* The **Orchestrator** module that sequences these steps as a workflow and manages the kick-off of the Data Prepper 
+process between them. The orchestrator module helps route the correct components when using Docker, though each component can be executed seperately using Python.
 
-The orchestrator module is the Docker entrypoint for the tool, though each component can be executed separately 
-via Python. Help text for each module can be printed by supplying the `-h / --help` flag.
+Help text for each module can be printed using the `-h / --help` flag when exectuing the Python script.
 
 ## Current Limitations
 
-* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing
-* The tool does not support customizing the list of indices included for migration
-* Metadata migration only supports basic auth
-* The migration does not filter out `red` indices
-* In the event that the migration fails or the process dies, the created indices on the target cluster are not rolled back
+The Fetch Migration currently has the following limitations:
+
+* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing.
+* Fetch Migration does not support customizing the list of indices included for migration.
+* Fetch migration only supports basic authentication.
+* The migration does not filter out indexes whose health is `red`.
+* In the event that the migration fails or the process dies, the created indexes on the target cluster are not rolled back to the previous version.
+
+## Running Fetch Migration
 
-## Execution
+Use either [Python](#python) or [Docker](#Docker) to run the Fetch Migration tool.
 
 ### Python
 
-* [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this GitHub repo
-* Install [Python](https://www.python.org/)
-* Ensure that [pip](https://pip.pypa.io/en/stable/installation/#) is installed
-* (Optional) Set up and activate a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-and-using-virtual-environments)
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the `opensearch-migrations` repo using `git clone git@github.com:opensearch-project/opensearch-migrations.git`.
+2. Install [Python](https://www.python.org/).
```

Comment:
Python3

Reply:
Isn't `pip` apart of the python3 package?